include/unifex/sender_concepts.hpp

```diff
@@ -356,6 +356,30 @@ template <typename Sender>
 constexpr bool is_sender_nofail_v =
     sender_error_types_t<Sender, is_empty_list>::value;
 
-} // namespace unifex
+/// \cond
+namespace _single_sender {
+template <typename... Types>
+using _is_single_valued_tuple = std::bool_constant<1 >= sizeof...(Types)>;
```

Comment:
Does `std::bool_constant<1 >= sizeof...(Types)>` need parentheses around bool expression to avoid it being parsed incorrectly?